chat-sdk/src/main/java/com/amazon/connect/chat/sdk/network/WebSocketManager.kt

```diff
@@ -322,6 +322,11 @@ class WebSocketManagerImpl @Inject constructor(
                             handleTyping(jsonObject, jsonString)
                         ContentType.ENDED ->
                             handleChatEnded(jsonObject, jsonString)
+                        ContentType.TRANSFER_SUCCEEDED, ContentType.TRANSFER_FAILED,
+                        ContentType.PARTICIPANT_IDLE, ContentType.PARTICIPANT_RETURNED,
+                        ContentType.PARTICIPANT_INVITED, ContentType.AUTO_DISCONNECTION,
+                        ContentType.CHAT_REHYDRATED ->
+                            createCommonEvent(jsonObject, jsonString)
```

Comment:
This is still creating two event objects.  Could we just store the value and then re-use it for these events?